CHANGELOG.md

```diff
@@ -38,57 +27,70 @@ If the extension is to be switched off, the following must now be set, as the ex
   - Catalog Crawler CE: `ghcr.io/sovity/catalog-crawler-ce:{{ VERSION }}`
 - Connector UI Docker Image: `ghcr.io/sovity/edc-ui:{{ UI VERSION }}`
 
-
-## [10.1.0] - 2024-08-09
+## [10.2.0] - 2024-08-20
 
 ### Overview
 
-MDS 2.2 patch release
+API Wrapper update and bug fixing, database performance improvements.
```

Comment:
Might want to fix wording.
Maybe "API Wrapper update, several bug fixes and database performance improvements"